in_text/text_00_parent.yml

```diff
@@ -31,7 +31,7 @@ process-date: !expr format(Sys.time(),'%Y%m%d')
 process-description: >-
     This is the landing page for the other data release sections; processing descriptions can be found on each child item.
 indirect-spatial: U.S.A.
-latitude-res: 0.1
+latitude-res: 0.1 
 longitude-res: 0.1
```

Comment:
Still curious about how resolution gets chosen for non-gridded data. I know that this was taken from last year's data release, but it would be useful to know where it originated.